ssl/quic/quic_port.c

```diff
@@ -814,9 +963,11 @@ static void port_send_retry(QUIC_PORT *port,
                             QUIC_PKT_HDR *client_hdr)
 {
     BIO_MSG msg[1];
-    unsigned char buffer[512], *token_buf = NULL;
+    unsigned char buffer[512];
+    unsigned char token_buf[MARSHALLED_TOKEN_MAX_LEN];
```

Comment:
The buffer could be reused for token_buf. But I assume we can optimize this later.

Reply:
If we do that, I guess we can just use a buffer size of 512 for both and not worry about `MARSHALLED_TOKEN_MAX_LEN`